Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: all all-tests test
+NIX_PATH := nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixpkgs-unstable.tar.gz
```

Comment:
We should pin this to whatever version of `nixpkgs-unstable` the `./format` script uses.

Reply:
Is this a good idea though? I ask this for two reasons:

- This may cause unwanted CI failures and format only because we are bumping the version used in `Makefile` or vice-versa
- Also, one of the reasons I think tracking a more up-to-date branch is better is to get issues happening in unstable faster. Like this one: https://github.com/nix-community/home-manager/issues/2317. Fixing a commit will hide those issues until someone bump the commit (that may cause the first issue again)

Reply:
Hmm... I guess that's fair. If this is only for tests, then I think it makes sense to track the latest tar file.